aspnetcore/fundamentals/http-requests/samples/3.x/HttpClientFactorySample/GitHub/GitHubBranch.cs

```diff
@@ -1,10 +1,13 @@
+using System.Text.Json.Serialization;
+
 namespace HttpClientFactorySample.GitHub
 {
     /// <summary>
     /// A partial representation of a branch object from the GitHub API
     /// </summary>
     public class GitHubBranch
     {
+        [JsonPropertyName("name")]
```

Comment:
It keeps the referenced sample code cleaner if we use these attributes to get the camel-case effect. Otherwise, the calls to `JsonSerializer.DeserializeAsync(...)` need to have a `JsonSerializerOptions` argument passed in and that bloats things a little. It's secondary to `HttpClient` really so I think this is a good compromise.